app/_src/production/cp-deployment/multi-zone.md

```diff
@@ -192,85 +192,105 @@ You need the following values to pass to each zone control plane setup:
 {% tabs zone-control-planes useUrlFragment=false %}
 {% tab zone-control-planes Kubernetes %}
 
-**Without zone egress**:
-
 1.  On each zone control plane, run:
 
+    {% if_version gte:2.3.x %}
     ```sh
     kumactl install control-plane \
     --mode=zone \
     --zone=<zone name> \
     --ingress-enabled \
-    --kds-global-address grpcs://<global-kds-address>:5685 | kubectl apply -f -
+    --kds-global-address grpcs://<global-kds-address>:5685 \
+    --set controlPlane.tls.kdsZoneClient.skipVerify=true | kubectl apply -f -
```

Comment:
This is not going to be ported well to KM where we need `kuma.` prefix. Same for all helm values.

Reply:
is it problem of this PR or all docs?

Btw. I thought we solved helm prefixes already...

Reply:
We do actually. Let me add this to the whole page `{{set_flag_values_prefix}}`